CONTRIBUTING.md

```diff
@@ -1,5 +1,74 @@
-# CONTRIBUTING
+# Contributing to Mistral AI Client for Dart
 
-## Development flow
+Thank you for considering contributing to [Your Project Name]! We appreciate your time and effort.
 
-## How to build
+## Table of Contents
+
+- [Contributing to Mistral AI Client for Dart](#contributing-to-mistral-ai-client-for-dart)
+  - [Table of Contents](#table-of-contents)
+  - [How Can I Contribute?](#how-can-i-contribute)
+    - [Reporting Bugs](#reporting-bugs)
+    - [Suggesting Enhancements](#suggesting-enhancements)
+    - [Code Contributions](#code-contributions)
+    - [Documentation](#documentation)
+  - [Getting Started](#getting-started)
+    - [Setting Up the Development Environment](#setting-up-the-development-environment)
+  - [Style Guide](#style-guide)
+  - [Review Process](#review-process)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+Before submitting a bug report, please check if the issue has already been reported in the [issue tracker](https://github.com/nomtek/mistralai_client_dart/issues). If it hasn't, create a new issue with a clear title and description, including steps to reproduce the bug.
+
```

Comment:
Maybe it is a good idea to add some  Issue templates(.md file), where we will show how bugs should be reported. 
Additionally, we will fulfill one of the best practices from insights-> community standards in our project.

Reply:
Addressed here https://github.com/nomtek/mistralai_client_dart/pull/25. I'm closing this conversation on your behalf @MarcinGruchala